lkmpg.tex

```diff
@@ -868,7 +868,7 @@ \subsection{Manage /proc file with seq\_file}
 \begin{itemize}
 \item \url{http://lwn.net/Articles/22355/}
 
-\item \url{http://www.kernelnewbies.org/documents/seq\_file\_howto.txt}
+\item \url{https://kernelnewbies.org/Documents/SeqFileHowTo}
```

Comment:
Don't do this! Submit another pull request for fix dead link(s).

Reply:
Sorry, I commit to the wrong branch accidently.